docs/GettingStarted.asciidoc

```diff
@@ -395,12 +395,13 @@ following command:
 
 [source,sh]
 --------------------------------------------------------------------------------
-/var/lib/openqa/tests/opensuse/templates
+/var/lib/openqa/tests/opensuse/templates --apikey [API_KEY] --apisecret [API_SECRET]
 --------------------------------------------------------------------------------
```

Comment:
as [] marks optional arguments, I guess the proper way is to write
[-apikey APIKEY]